server/start.go

```diff
@@ -230,7 +231,8 @@ func startInProcess(ctx *Context, cdc codec.JSONMarshaler, appCreator types.AppC
 		clientCtx := client.Context{}.
 			WithHomeDir(home).
 			WithChainID(genDoc.ChainID).
-			WithJSONMarshaler(cdc).
+			WithJSONMarshaler(legacyAminoCdc).
```

Comment:
Why do we need to use the legacy amino codec here? Is it for legacy querier support?

Reply:
Yes

Reply:
Comment added